torchtext/csrc/register_bindings.cpp

```diff
@@ -8,17 +8,6 @@
 #include <vectors.h>
 #include <vocab.h>
 
-// If we are in a Windows environment, we need to define
```

Comment:
Why do we remove this?

Reply:
I was just testing to see if the registration had anything to do with the error. Readded this

Reply:
It is okay to remove this registration now because it is not a Python extension anymore. But I'm curious about the crash.

Reply:
Hey @peterjc123 I removed the python Windows init and now we are getting different test failures. Could you take another look?